lib/vpn/vpn_tab.dart

```diff
@@ -7,13 +8,43 @@ import 'vpn_server_location.dart';
 import 'vpn_status.dart';
 import 'vpn_switch.dart';
 
-class VPNTab extends StatelessWidget {
+class VPNTab extends StatefulWidget {
   const VPNTab({Key? key}) : super(key: key);
 
+  @override
+  _VPNTabState createState() => _VPNTabState();
+}
+
+class _VPNTabState extends State<VPNTab> {
+  WebsocketImpl websocket = WebsocketImpl();
+
+  @override
+  void initState() {
+    if (isDesktop()) {
+      connectDesktopSocket();
```

Comment:
@atavism Is there any way that we can create a socket connection only and use the same connection access app?

Reply:
That's how it currently works

Reply:
Thanks for changes @atavism.